### ISSUE TITLE
SA1629 reports false positive when before-last inline child is xml element

### ISSUE THREAD

**Issue by generateui** [DotNetAnalyzers/StyleCopAnalyzers#3537]
When the content of the summary before the dot character is an inline xml element, a false positive is reported. Example:

```csharp
/// <summary>
/// Id of the team member that deleted the member specified in
/// <see cref="DeletedMemberId">.
/// </summary>
```
I expect above exmaple code not to report a warning, but it does.

**Comment by sharwell** [comment]
This test appears to be verifying that there is no diagnostic reported in this case:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/6478404520b39f002ae0ce2d062f1474cf27b14f/StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1629UnitTests.cs#L351-L362

**Comment by bjornhellander** [comment]
Your comment has a syntax error (triggering CS1570) due to a missing slash in the `see `element. If you correct the comment, then SA1629 goes away as well.
```
/// <summary>
/// Id of the team member that deleted the member specified in
/// <see cref="DeletedMemberId"/>.
/// </summary>
```
